tests/rptest/tests/flink_basic_test.py

```diff
@@ -170,7 +170,6 @@ def test_basic_workload(self):
 
         return
 
-    @ok_to_fail
```

Comment:
nit: this file changes belong to the previous commit?

Reply:
No, this is a part of dev branch, which has this test marked so that it will not prevent us from releasing stuff. Since this test is ready and fixed. I am removing the mark.